src/Sylius/Behat/Context/Ui/Shop/AddressBookContext.php

```diff
@@ -210,6 +210,16 @@ public function itShouldContain($value)
         Assert::true($this->addressBookIndexPage->addressOfContains($fullName, $value));
     }
 
+    /**
+     * @Then /^it should contain country "([^"]+)"$/
```

Comment:
By `it` in the sentence I would assume, that the current address is already fetched. Can we take last response from the client instead of making a new request?